src/v2/mod.rs

```diff
@@ -2071,6 +2079,24 @@ impl Client {
         Ok(node_info)
     }
 
+    /// Get the projected earliest time a baker wins the opportunity to bake a
+    /// block.
+    /// If the baker is not a baker for the current reward period then then the
+    /// timestamp returned is the projected time of the first block of the
+    /// new reward period.
+    /// Note that the endpoint is only available on a node running consensus
+    /// version 1, if the node is running consensus version 0, then an error
+    /// will be returned.
```

Comment:
"Consensus version" isn't really used in the SDK or public API much. Better to use protocol versions as well.